featuremanagement/_featuremanager.py

```diff
@@ -99,7 +99,7 @@ def is_enabled(self, feature_flag_id, **kwargs):
             filter_name = feature_filter[FEATURE_FILTER_NAME]
             if filter_name in self._filters:
                 if feature_conditions.requirement_type == REQUIREMENT_TYPE_ALL:
-                    if not self._filters[filter_name].evaluate(feature_filter[FEATURE_FILTER_PARAMETERS], **kwargs):
+                    if not self._filters[filter_name].evaluate(feature_filter, **kwargs):
```

Comment:
@zhiyuanliang-ms, thanks for catching this. Can you update the async version too, so I don't have to do it in a different PR?

Reply:
@mrm9084 No problem.